Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Dockerfile that installs imap_processing and its dependencies
+FROM public.ecr.aws/docker/library/python:3.10-slim
+USER root
+
+# Location for Core package installation location.
+ENV IMAP_PROCESS_DIRECTORY=/opt/imap
+WORKDIR $IMAP_PROCESS_DIRECTORY
+
+# Turn off interactive shell to suppress configuration errors
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install git
+RUN apt-get update && apt-get install -y git
+
+# Upgrade pip to the latest version
+RUN pip install --upgrade pip
+
+# TODO: install imap_processing instead of copying it once a new version is released
+# Install imap_processing
+#RUN pip install imap_processing
+# TODO: delete this section once new version released for imap_processing
+RUN pip install git+https://github.com/IMAP-Science-Operations-Center/imap_processing.git@dev
+
+# Copy over only the necessary scripts
+COPY imap_processing/run_processing.py $IMAP_PROCESS_DIRECTORY/run_processing.py
+COPY tools $IMAP_PROCESS_DIRECTORY/tools
```

Comment:
Do we need to copy `tools` folder? I think it's only used locally to create XTCE.

Reply:
I removed the example I had of how to use the kernels, which is what I was using the tools folder for. But I removed the example.